hal/src/main/java/edu/wpi/first/hal/DriverStationJNI.java

```diff
@@ -125,7 +125,7 @@ public static native int sendError(
 
   public static native int sendConsoleLine(String line);
 
-  public static native void refreshDSData();
+  public static native boolean refreshDSData();
```

Comment:
It doesn't seem like any of the boolean return values here are used?  Is this just to help make future debugging easier?

Reply:
Correct. And it could be used by teams in custom templates if they wanted to know if there was new data for the current iteration, without needing the whole loop thing.